.vscode/sftp.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Colmena Project",
```

Comment:
COLMENA Manager is the true name

Reply:
I consider that vscode files should not be included in the repo, so i added it to gitignore in one of the more recent commits